dev/NavigationView/NavigationViewList.h

```diff
@@ -30,6 +31,13 @@ class NavigationViewList :
     // IControlOverrides / IControlOverridesHelper
     void OnKeyDown(winrt::KeyRoutedEventArgs const& e);
 
+    winrt::com_ptr<ViewModel> ListViewModel() const;
+    void ListViewModel(winrt::com_ptr<ViewModel> viewModel);
+
+    winrt::TreeViewNode NodeAtFlatIndex(int index) const;
+    winrt::TreeViewNode NodeFromContainer(winrt::DependencyObject const& container);
+    winrt::DependencyObject ContainerFromNode(winrt::TreeViewNode const& node);
+
```

Comment:
I expect it share the same code with treeviewlist. maybe we can define a base class like XXViewBase, the both NavigationViewList and TreeViewList as a subclass of it.